src/containers/App/App.js

```diff
@@ -1,37 +1,34 @@
 import React, { Component } from 'react';
+import { Switch, Route, withRouter } from 'react-router-dom';
 import Header from '../Header/Header';
-import Palette from '../Palette/Palette';
-import ControlForm from '../ControlForm/ControlForm';
-import ProjectContainer from '../ProjectContainer/ProjectContainer';
 import '../../main.scss';
-import { connect } from 'react-redux';
-import { fetchProjects } from '../../thunks/fetchProjects';
+import Share from '../Share/Share';
+import Home from '../Home/Home';
+import NotFound from '../../components/NotFound/NotFound';
 
 export class App extends Component {
```

Comment:
This can be refactored to a functional component now that we don't need access to componentDidMount()